index.html

```diff
@@ -114,6 +114,31 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
+        
+        <!-- ________ *Rafael* Contributor card START ________  -->
+      <div class="card">
+        <p class="name">Rafael José Jiménez</p>
+        <p class="contact">
+          <i class="fab fa-twitter"></i>
+          <a href="https://twitter.com/fistan85" target="_blank">@fistan85</a>
+        </p>
+        <p class="about">...</p>
```

Comment:
Kindly added about info
```suggestion
        <p class="about">Your about info here...</p>
```